checker/jtreg/nullness/issue824/Class2.java

```diff
@@ -1,9 +1,10 @@
 /*
  * @test
  * @summary Test case for Issue 824 https://github.com/typetools/checker-framework/issues/824
+ * The defaults for type variable upper bounds with type Object changed since the issue was filed.  So, this test case has been changed so that annotations on type variable bounds in stub files is still tested.
```

Comment:
I found the button; it was on the red X for "required code review" at the bottom of the page where the CI test results also appear.